plugin/vscode/quick-lint-js/vscode-error-reporter.h

```diff
@@ -60,7 +60,22 @@ class vscode_error_formatter
             this->env_, reinterpret_cast<const char*>(this->message_.data())),
         /*severity=*/severity,
     });
-    diag.Set("code", ::Napi::String::New(this->env_, code.data(), code.size()));
+
+    char error_path[14];
+    sprintf(error_path, "/errors/%s", code.data());
+
+    ::Napi::Object uri = ::Napi::Object::New(this->env_);
+    uri.Set("scheme", ::Napi::String::New(this->env_, "https"));
+    uri.Set("authority", ::Napi::String::New(this->env_, "quick-lint-js.com"));
+    uri.Set("path", ::Napi::String::New(this->env_, error_path));
+    uri.Set("fragment", ::Napi::String::New(this->env_, ""));
+    uri.Set("query", ::Napi::String::New(this->env_, ""));
```

Comment:
Must fix: The TypeScript definitions say that the `target` property must be an instance of `Uri`. A new object with the default prototype is not an instance of `Uri`. We should adhere to the documented specification:
```typescript
    export class Diagnostic {
        code?: string | number | {
            value: string | number;
            target: Uri;
        };

        // ...
      }
```